### DISCUSSION TITLE
v3.2.0

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#831] [Announcements]
## Changes

- Fix multipacket mqtt receives @DTTerastar (#830)
- Use less memory @DTTerastar (#820)
- AHTX0: fix receive state of temperature and humidity @niceboygithub (#810)
- Add a configuration option for the Home Assistant Discovery Prefix @jlrgraham (#793)
- Add new setting to adjust for weak antennas @DTTerastar (#785)
- Miflora fingerprinting @DTTerastar (#781)
- Fix advertising interval @DTTerastar (#769)
- Advertise every 2-4s @DTTerastar (#768)
- Enable calibrating internal iBeacon ref rssi @DTTerastar (#767)
- Add nicer ui after pressing Enroll @DTTerastar (#765)
- Added support for AHT10/AHT20 temperature + humidity sensors @URKidding (#761)
- Fix Typo In Radar Pin Configuration @DTTerastar (#764)
- Have nodes broadcast an ibeacon advertisement @DTTerastar (#763)
- previous pull incomplete @mjtimmer (#737)

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v3.2.0'>v3.2.0</a>.</em>

**Comment by Wheemer** [comment]
Thanks for the work, I'm sure I speak for everyone when I say we appreciate your efforts.
